operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/url/DownloadUrl.java

```diff
@@ -43,9 +43,10 @@
     private String secretKey = null;
     private boolean parquet = false;
     private boolean presigned = false;
+    private boolean path = true;
     private ZonedDateTime time = ZonedDateTime.now(ZoneOffset.UTC);
 
-    public DownloadUrl(String server, String region, String bucketName, String cosBucketPathPrefix, String objectKey, String accessKey, String secretKey, boolean parquet, boolean presigned) {
+    public DownloadUrl(String server, String region, String bucketName, String cosBucketPathPrefix, String objectKey, String accessKey, String secretKey, boolean parquet, boolean presigned, boolean path) {
```

Comment:
to me, it would be more clear to use the S3HostStyle enum vs a boolean.
if we want to stick with boolean, I'd name it "pathBasedHost" or something (and add javadoc).

Reply:
changed